README.md

```diff
@@ -20,3 +20,84 @@ npm run build
 ```
 
 # [Reference application](https://github.com/Kitware/candela/blob/master/app/resonant-reference-app/README.md)
+
+# Release Process
+
+To perform a new release of Candela, please follow these steps. This assumes
+that the code on ``master`` is ready to be turned into a new release (i.e., it
+passes all tests and includes all new functionality desired for the new
+release). In this example, we will pretend that the new release version number
+will be 1.2.0.
+
+1. Create a new release branch, named ``release-1.2.0``:
+
+   ```` 
+   git checkout -b release-1.2.0 master 
+   ````
+
+2. Bump the version number to 1.2.0 by editing ``package.json``. Make a commit
+   with the commit message "Bump version number for release" and push the
+   branch:
+
+   ````
+   vim package.json
+   git commit -am 'Bump version number for release'
+   git push -u origin release-1.2.0
+   ````
+
+3. Make a new local branch to save your spot in the commit tree here.
```

Comment:
Since I've never used a bare `branch` command like this to make a branch but not switch to it (I always use `git checkout -b save-point`, perhaps add a little clarifying logic like "Be sure your checkout remains on the release branch. You can do this with:".